langserver/lsp.go

```diff
@@ -0,0 +1,95 @@
+package langserver
+
+type InitializeParams struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type InitializeParams should have comment or be unexported